qualification/report/katdoc.sty

```diff
@@ -103,147 +123,121 @@
           \hline
         \end{tabularx}
       \end{table}
+    \end{center}
+  \end{titlepage}
+  %\stepcounter{page}
```

Comment:
I see previously there were commented-out lines of code/LaTeX still in the file (mainly `\vspace`), is this left in mostly as a reminder?